### ISSUE TITLE
Using Stoneship Chart Room during discard phase

### ISSUE THREAD

**Issue by rubenpieters** [mtgred/netrunner#6591]
There is a paid ability window after the discard step, but before end-of-turn/discard triggers are met (5.7.2b). This means that you are allowed to use Stoneship Chart Room before checking for Lat's ability. Currently, this needs to be resolved manually.

**Comment by NoahTheDuke** [comment]
Yeah we don't have paid ability windows, so until we do, this will have to be manual.

**Comment by NBKelly** [comment]
See also: #5888
